src/Microsoft.DotNet.ScenarioTests.SdkTemplateTests/DotNetSdkHelper.cs

```diff
@@ -262,6 +262,66 @@ public void ExecuteAddClassReference(string projectDirectory)
         ExecuteCmd($"add reference {classDirectory}", projectDirectory);
     }
 
+    public void ExecuteWorkloadInstall(string projectDirectory, string templates)
```

Comment:
Suggestion: Rename the parameter to workloadIds

Reply:
Fixed in latest commit